src/lib/polymer-bootstrap.html

```diff
@@ -37,13 +37,28 @@
       var options = {
         prototype: prototype
       };
-      // NOTE: we're specifically supporting older Chrome versions here 
+      // NOTE: we're specifically supporting older Chrome versions here
       // (specifically Chrome 39) that throw when options.extends is undefined.
       if (prototype.extends) {
         options.extends = prototype.extends;
       }
-      Polymer.telemetry._registrate(prototype);
-      document.registerElement(prototype.is, options);
+
+      try {
```

Comment:
Yep. Until then, my point was more that this piece of code gets executed very few times. We have other hot paths in polymer core (`endOfMicroTask`) that are deptop'd due to try/catch that should be looked at first.